applications/CoSimulationApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -54,6 +55,12 @@ namespace Python{
             .value("Origin", FetiSolverIndexType::Origin)
             .value("Destination", FetiSolverIndexType::Destination)
             ;
+
+        pybind11::class_< ConversionUtilities>(m, "ConversionUtilities")
+            .def("ConvertElementalDataToNodalData",
```

Comment:
```suggestion
            .def_static("ConvertElementalDataToNodalData",
```

Reply:
ohh ..Fine . As it is static function, hence we need to do like this. Got it. Thanks :))